### PR TITLE
chore: updates zone details and subscriptions

### PR DESCRIPTION
**Pull request by kleinfreund**
**feat: only queries envoy data if resource is online**

Changes EnvoyData to only query envoy data if the corresponding resource is online.

**chore(zones): moves subscriptions into overview tab for ingress/egress details**

**chore(subscriptions): collapses responses sent and acknowledged, adds total**

Collapses the responses sent and acknowledged entries.

Adds the total number of responses sent and acknowledged.

Renames the "Insights" tab to "Subscriptions".

Signed-off-by: Philipp Rudloff <philipp.rudloff@konghq.com>

### CODE REVIEW

**Comment by kleinfreund** [comment]
> The allignments and signing is a little weird on a 15" laptop:

We could change "Connect time" to "Connected", "Disconnect time" to "Disconnected", and "Responses sent/acknowledged" to "Responses". If that’s not enough to make this nicer on smaller screens, we need to reconsider what to show there and what we can possible move (e.g. we could consider moving connected/disconnected into the collapsed part and only show "Connected: {timestamp}" on the very first (i.e. last) subscription).

**Comment by lahabana** [comment]
Connected/Disconnected and `Responses sent/acked` sounds good enough. Also do you know why the first line is indented differently?

**Comment by kleinfreund** [comment]
> Connected/Disconnected and `Responses sent/acked` sounds good enough. Also do you know why the first line is indented differently?

Changed both now.

File: src/app/common/subscriptions/SubscriptionHeader.vue [hunk 1]
```diff
@@ -47,16 +53,34 @@ const globalInstanceId = computed(() => 'globalInstanceId' in props.subscription
...
 
 <style lang="scss" scoped>
 .subscription-header {
   display: grid;
-  grid-template-columns: repeat(3, 1fr);
+  grid-template-columns: 30ch repeat(3, 1fr);
+  gap: $kui-space-80;
   padding-right: $kui-space-40;
   padding-left: $kui-space-40;
   font-size: 1.125rem;
```
[the remark this answers is not in the document]
Comment (kleinfreund) [reply src/app/common/subscriptions/SubscriptionHeader.vue:74]: Yeah, good idea. I don’t mind removing this. The hierarchical nature of the data presentation still works without.

**Review by lahabana — APPROVED** [review]

**Pull request merged**